packages/ember-toucan-form/src/-private/multiselect-field.hbs

```diff
@@ -25,6 +25,7 @@
       @isDisabled={{@isDisabled}}
       @isReadOnly={{@isReadOnly}}
       @label={{@label}}
+      @noResultsText='No results'
```

Comment:
Same questions here - I don't think we'll want to set a default that can't be overridden due to intl reasons.